include/albatross/src/models/sparse_gp.hpp

```diff
@@ -317,7 +341,9 @@ class SparseGaussianProcessRegression
       B.col(pi).topRows(i + 1) = old_fit.sigma_R.col(i).topRows(i + 1);
     }
     B.bottomRows(n_new) = A_ldlt.sqrt_solve(K_fu);
-    const auto B_qr = B.colPivHouseholderQr();
+    auto B_qr = QRImplementation::compute(
+        B, Base::threads_
+               .get()); // SPQR_create(B.sparseView(), Base::threads_.get());
```

Comment:
nit: remove comment. Should `B_qr` be `const`?

Reply:
Done.